roles/download/tasks/prep_download.yml

```diff
@@ -18,6 +18,15 @@
     image_info_command: "{{ bin_dir }}/nerdctl -n k8s.io images --format '{% raw %}{{ '{{' }} .Repository {{ '}}' }}:{{ '{{' }} .Tag {{ '}}' }}{% endraw %}' 2>/dev/null | grep -v ^:$ | tr '\n' ','"
     image_pull_command: "{{ bin_dir }}/nerdctl -n k8s.io pull --quiet"
```

Comment:
I would use a conditional on this line instead of adding another task, it's less code to maintain.

Reply:
thanks for the advice, I tried using conditional code, How about this?